com.valvesoftware.Steam.yml

```diff
@@ -57,6 +57,7 @@ finish-args:
   - --env=PYTHONPATH=/app/utils/lib/python3.8/site-packages
   - --env=PROTON_DEBUG_DIR=/var/tmp
   - --env=XDG_CONFIG_DIRS=/etc/xdg:/usr/lib/x86_64-linux-gnu/GL:/usr/lib/i386-linux-gnu/GL
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/app/lib32/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0
```

Comment:
Maybe we should remove `/app/lib{,32}/gstreamer-1.0` from the list, since we don't bundle any gstreamer plugins?

Reply:
I'm fine with it as is. We can leave it as is so it will work sanely if we ever bundle any plugins.